### ISSUE TITLE
Can escape border with bed

### ISSUE THREAD

**Issue by Projecki** [BentoBoxWorld/Border#129]
### Expected behavior

Unable to escape border

### Observed/Actual behavior

Can escape border by placing a bed at the edge, sleeping and waking up on the other side

### Steps/models to reproduce

https://youtu.be/a7IKCt7VshI

### BentoBox version

[18:25:20 INFO]: Running Invalid (UNIVERSESPIGOT) 1.20.4.
[18:25:20 INFO]: (git-UniverseSpigot-"1954a51" (MC: 1.20.4))
[18:25:20 INFO]: BentoBox version: 2.3.0
[18:25:20 INFO]: Database: MARIADB
[18:25:20 INFO]: Loaded Game Worlds:
[18:25:20 INFO]: boxed_world (Boxed): Overworld, Nether
[18:25:20 INFO]: Loaded Addons:
[18:25:20 INFO]: Border 4.3.0 (ENABLED)
[18:25:20 INFO]: Boxed 2.5.0 (ENABLED)

**Comment by tastybento** [comment]
Thanks. Yes, this is actually possible with world borders in general, however, if you have:

```
return-teleport: true
```
in the border config, as soon as the player moves, they should be teleported back to inside the box.

**Comment by Projecki** [comment]
I actually do already have return-teleport set to true

**Comment by tastybento** [comment]
Okay. So, it works?

**Comment by Projecki** [comment]
Correct, even with return-teleport set to true, players are able to escape their border on my server

**Comment by tastybento** [comment]
No, I mean, I can't prevent the initial escape, but if they move after escaping then they will be teleported back to within their box. So long as that is happening, it's all working.

**Comment by Projecki** [comment]
![image](https://github.com/BentoBoxWorld/Border/assets/65093423/7eaf198a-0976-43c4-a7ea-e8a178994317)

right, they can escape and they can even build, but no break, outside the border.

I haven't done any testing on a server with only BentoBox installed, though. Haven't had time to properly test it out

**Comment by tastybento** [comment]
Hmm that shouldn't be possible. I tested it and I get teleported back into the border. Investigate a bit more and let me know how it goes.
